CodeComplianceTest_Engine/Compute/CheckProjectFile.cs

```diff
@@ -108,15 +108,15 @@ private static TestResult CheckNETTarget(this ProjectFile csProject, List<string
                 {
                     string fullXMLText = $"<TargetFramework>{target}</TargetFramework>";
                     int lineNumber = fileLines.IndexOf(fileLines.Where(x => x.Contains(fullXMLText)).FirstOrDefault()) + 1; //+1 because index is 0 based but line numbers start at 1 for the spans
-                    finalResult = finalResult.Merge(Create.TestResult(TestStatus.Warning, new List<Error> { Create.Error($"Target frameworks for BHoM projects should either be .Net Framework 4.7.2, .Net Framework 4.8, .Net Framework 4.8.1, .Net Standard 2.0, .Net 5.0, or .Net 6.0.", Create.Location(csProjFilePath, Create.LineSpan(lineNumber, lineNumber)), documentationLink, TestStatus.Warning) }));
+                    finalResult = finalResult.Merge(Create.TestResult(TestStatus.Warning, new List<Error> { Create.Error($"Target frameworks for BHoM projects should either be .Net Framework 4.7.2, .Net Framework 4.8, .Net Framework 4.8.1, .Net Standard 2.0 or, .Net 5.0 to 9.0.", Create.Location(csProjFilePath, Create.LineSpan(lineNumber, lineNumber)), documentationLink, TestStatus.Warning) }));
                 }
                 else
                     atLeastOneCorrect = true;
             }
 
             if(!atLeastOneCorrect)
             {
-                finalResult = finalResult.Merge(Create.TestResult(TestStatus.Error, new List<Error> { Create.Error($"At least one of the Target frameworks for BHoM projects must either be .Net Framework 4.7.2, .Net Standard 2.0, or .Net 5.0.", Create.Location(csProjFilePath, Create.LineSpan(1, 1)), documentationLink) }));
+                finalResult = finalResult.Merge(Create.TestResult(TestStatus.Error, new List<Error> { Create.Error($"At least one of the Target frameworks for BHoM projects must either be .Net Framework 4.7.2, .Net Standard 2.0, or .Net 5.0 to 9.0.", Create.Location(csProjFilePath, Create.LineSpan(1, 1)), documentationLink) }));
```

Comment:
While you are changing this, can you also update this line to include `.Net Framework 4.8, .Net Framework 4.8.1,` as the one above does?